src/structures/Chat.js

```diff
@@ -57,6 +57,12 @@ class Chat extends Base {
          */
         this.archived = data.archive;
 
+        /**
+         * Indicates if the Chat is muted
+         * @type {boolean}
+         */
+        this.isMuted = data.mute.isMuted;
```

Comment:
Not sure if this works. But in the browser `mute` is definitely an object.
I forgot if `data` is serialized or not.